soroban-sdk/src/vec.rs

```diff
@@ -563,6 +564,66 @@ where
     }
 }
 
+impl<T> Vec<T>
+where
+    for<'a> &'a T: IntoVal<Env, RawVal>,
+{
+    /// Returns true if the Vec contains the item.
+    #[inline(always)]
+    pub fn contains(&self, item: impl Borrow<T>) -> bool {
```

Comment:
@jonjove @graydon I'm interested in your feedback on the interface I'm using for these functions where I'm accepting a `Borrow<T>` so that people can pass a `&T` or a `T`. I saw there is a more complicated use of this in the docs at https://doc.rust-lang.org/std/borrow/trait.Borrow.html, but it is unclear to me if we need to go to that length.

Reply:
I'm good with this.